bigbluebutton/bigbluebutton-html-plugin-sdk
# feat(html5): Added support for persistEvent

### What does this PR do?

This PR implemented the `pluginApi.persistEvent` function just as requested by https://github.com/bigbluebutton/bigbluebutton-html-plugin-sdk/issues/121.


### Closes Issue(s)
Closes https://github.com/bigbluebutton/bigbluebutton-html-plugin-sdk/issues/121

### How to test
Create a meeting with `record=true` to capture `events.xml` and with the following parameter:
```
pluginManifests=[{"url":"http://172.17.0.1:4701/manifest.json"}]
```
run the sample-use-meeting plugin with an  `npm start` and with the correct SDK.

Wait for a few seconds, end the meeting and check the `events.xml` by doing the folloing commands:

```bash
sudo updatedb
vi $(locate events.xml | grep <meeting-id>)
```

The meeting id can be found with `/getMeetingInfo` endpoint in the ApiMate for instance.

### More
Closely related to https://github.com/bigbluebutton/bigbluebutton/pull/21389
